### PR TITLE
update icon: bitbucket (original, original-wordmark)

### PR DESCRIPTION
**Pull request by jusuchin85**
# **New icon section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [ ] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-) - `not required as this is an update to an existing icon`

## Link to prove your SVG is correct and up-to-date
https://atlassian.design/resources/logo-library/

### CODE REVIEW

**Comment by Panquesito7** [comment]
@Thomas-Boi, any idea why it looks like that? 👀 Thanks.

**Comment by jusuchin85** [comment]
@Thomas-Boi @Panquesito7 - updated PR with new commits. Hopefully, this will fix the icon text appropriately. 😃

File: icons/bitbucket/bitbucket-original-wordmark.svg [hunk 1]
```diff
@@ -1 +1,102 @@
...
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   width="32"
+   height="32"
+   viewBox="0 0 32 32"
```
Comment (Thomas-Boi) [inline icons/bitbucket/bitbucket-original-wordmark.svg:5]:
```suggestion
   viewBox="0 0 128 128"
```

File: icons/bitbucket/bitbucket-original.svg [hunk 2]
```diff
@@ -1 +1,10 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><g fill="#205081"><path d="M64 1.023v.002-.002c-30 0-54.557 8.074-54.557 18.1 0 2.64 6.55 40.504 9.147 55.52 1.165 6.733 18.56 16.607 45.39 16.607l.02-.08v.08c27 0 44.238-9.874 45.403-16.61 2.598-15.015 9.15-52.878 9.15-55.518C118.554 9.097 94 1.022 64 1.022zm0 78.062c-9.578 0-17.343-7.766-17.343-17.343C46.657 52.164 54.422 44.4 64 44.4s17.343 7.764 17.343 17.343c0 9.577-7.765 17.342-17.343 17.342zm-.01-54.368c-19.296-.03-34.934-3.384-34.928-7.49.007-4.107 15.656-7.41 34.952-7.378 19.296.03 34.934 3.382 34.927 7.49-.005 4.106-15.653 7.407-34.95 7.377z"/><path d="M103.212 89.69c-.83 0-1.494.588-1.494.588S88.283 100.918 64 100.918s-37.718-10.64-37.718-10.64-.665-.587-1.494-.587c-.99 0-1.93.666-1.93 2.135 0 .155.015.31.044.462 2.085 11.16 3.61 19.09 3.876 20.296 1.82 8.21 17.88 14.417 37.22 14.417H64c19.342 0 35.402-6.207 37.222-14.417.268-1.205 1.79-9.13 3.876-20.29.028-.153.044-.31.044-.465 0-1.47-.94-2.137-1.93-2.137z"/><circle cx="63.983" cy="61.737" r="8.691"/></g></svg>
\ No newline at end of file
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Thomas-Boi) [inline icons/bitbucket/bitbucket-original.svg:1]:
```suggestion
<svg viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg">
```

**Review by Thomas-Boi — CHANGES_REQUESTED** [review]
Just need to change the viewBox. Everything else looks good to me.

File: icons/bitbucket/bitbucket-original.svg (hunk 2 above)
Comment (jusuchin85) [reply icons/bitbucket/bitbucket-original.svg]: Updated!

File: icons/bitbucket/bitbucket-original-wordmark.svg (hunk 1 above)
Comment (jusuchin85) [reply icons/bitbucket/bitbucket-original-wordmark.svg]: Updated!

**Comment by Thomas-Boi** [comment]
@jusuchin85 everything looks good to me. Thank you very much for the PR. I'll leave this PR open temporary to test a script upgrade. Once that's tested and no other errors show up, I'll merge the PR 👍.

@Panquesito7 please review and merge [this PR](https://github.com/devicons/devicon/pull/1060) first 😄. Forgot to do something so this PR will fix it.

**Comment by jusuchin85** [comment]
Thanks @Thomas-Boi ! 😄

**Review by Thomas-Boi — CHANGES_REQUESTED** [review]
Hi @jusuchin85,

I'm sorry but I made a mistake. When resizing an SVG, you'd have to use a resizer, not just changing the viewBox.

For each of the icon, please change your viewBox back to `0 0 32 32` and resize it using this [website](https://www.iloveimg.com/resize-image/resize-svg#resize-options,pixels). Thank you for your patience and I'm sorry for the inconvenience.

**Comment by jusuchin85** [comment]
Icons resized accordingly using [iloveimg](https://www.iloveimg.com/resize-image/resize-svg#resize-options,pixels). Thanks for pointing this out, @Thomas-Boi !

File: icons/bitbucket/bitbucket-original-wordmark.svg [hunk 3]
```diff
@@ -1 +1,22 @@
...
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128px" height="px" viewBox="0 0 128 128" version="1.1">
```
Comment (Thomas-Boi) [inline icons/bitbucket/bitbucket-original-wordmark.svg:2]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128px" height="128px" viewBox="0 0 128 128" version="1.1">
```

**Review by Thomas-Boi — DISMISSED** [review]
Just need to change the values here. You can just press the Commit Suggestion button for quick fixes like this. No need to open your editor, change the files etc.

File: icons/bitbucket/bitbucket-original.svg [hunk 4]
```diff
@@ -1 +1,13 @@
...
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128px" height="px" viewBox="0 0 128 128" version="1.1">
```
[the remark this answers is not in the document]
Comment (Thomas-Boi) [reply icons/bitbucket/bitbucket-original.svg:2]: No worries Pan. The `height` attr will be removed by our SVG optimizer later.

**Review by Thomas-Boi — APPROVED** [review]
Looks good now! Thanks for your work and patience @jusuchin85

**Comment by jusuchin85** [comment]
No worries! Happy to help!

**Pull request merged**